src/routing/approved.rs

```diff
@@ -1564,26 +1566,39 @@ impl Approved {
     // Message sending
     ////////////////////////////////////////////////////////////////////////////
 
-    // Send NodeApproval to the current candidate which makes them a section member
+    // Send NodeApproval to a joining node which makes them a section member
     fn send_node_approval(
         &self,
-        peer: &Peer,
+        member_info: Proven<MemberInfo>,
         their_knowledge: Option<bls::PublicKey>,
     ) -> Result<Command> {
         info!(
-            "Our section with {:?} has approved candidate {:?}.",
+            "Our section with {:?} has approved peer {:?}.",
             self.section.prefix(),
-            peer
+            member_info.value.peer
         );
 
-        let their_knowledge = their_knowledge.and_then(|key| self.section.chain().index_of(&key));
-        let proof_chain = self
+        let addr = *member_info.value.peer.addr();
+
+        // Attach proof chain that includes the key the approved node knows (if any), the key its
```

Comment:
what does `the key its` mean ?

Reply:
it continues on the next line: "...the key its MembersInfo is signed with..."